client/src/containers/HomePage.tsx

```diff
@@ -100,12 +101,12 @@ class HomePage extends Component<HomePageProps, State> {
         ...searchAddress,
         housenumber: searchAddress.housenumber || "",
       })
-        .then((results) => {
+        .then((results: SearchResults) => {
           this.setState({
             results: results,
           });
         })
-        .catch((err) => {
+        .catch((err: any) => {
```

Comment:
We seem to type other "error" objects as type any (see `handleFormSubmit` on the Homepage), so I thought I'd stick to that here for consistency, but maybe something we want to revisit.